tutorials/asr/cloud/aws/SageMaker_ASR_Fine-tuning-CommonVoice.ipynb

```diff
@@ -0,0 +1,666 @@
+    "\"\"\"\n",
+    "# exit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61c4fbe2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install sagemaker awscli"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "876f553d",
+   "metadata": {},
+   "source": [
+    "### 1. Setup SageMaker with AWS Credentials\n",
+    "\n",
+    "If you haven't setup your AWS credentials, you can setup using the AWS CLI.\n",
+    "You will need your access and Secret key, with permissions to use SageMaker and S3."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1328482d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!aws configure"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01477d55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext tensorboard\n",
+    "from pathlib import Path\n",
+    "import os\n",
+    "import sagemaker\n",
+    "import wget\n",
+    "from omegaconf import OmegaConf\n",
+    "from sagemaker import get_execution_role\n",
+    "from sagemaker.pytorch import PyTorch\n",
+    "\n",
+    "from nemo.utils.notebook_utils import download_an4"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "405806f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sess = sagemaker.Session()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d099a96",
+   "metadata": {},
+   "source": [
+    "### 2. Download the NeMo source code\n",
+    "\n",
+    "SageMaker allows you to pass in your own source code, with an entrypoint script.\n",
+    "\n",
+    "Below we download the AWS NeMo `config.yaml` which contains our configuration, and the `speech_to_text_ctc_bpe.py` script to run training.\n",
+    "\n",
+    "Our folder structure will look like this:\n",
+    "\n",
+    "    code/\n",
+    "        speech_to_text_ctc_bpe.py\n",
+    "        conf/\n",
+    "            config.yaml\n",
+    "    data/\n",
+    "        manifests/\n",
+    "        raw_data/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b456c57",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "root_dir = Path('./')\n",
+    "code_dir = root_dir / 'code/'\n",
+    "config_dir = code_dir / 'conf/'\n",
+    "data_dir = root_dir / 'data/'\n",
+    "\n",
+    "root_dir.mkdir(exist_ok=True)\n",
+    "code_dir.mkdir(exist_ok=True)\n",
+    "config_dir.mkdir(exist_ok=True)\n",
+    "data_dir.mkdir(exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "721c7b7d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config_path = str(config_dir / \"config.yaml\")\n",
+    "wget.download(\"https://raw.githubusercontent.com/andrusenkoau/NeMo/esperanto_example/docs/source/asr/examples/esperanto_asr/scripts/get_commonvoice_data_v2.py\", str(code_dir))\n",
```

Comment:
It's downloading from a user. How about putting this script somewhere Nemo repo ? Or merge it with NeMo CV script